packages/builder/src/internal/context.ts

```diff
@@ -0,0 +1,65 @@
+import { BuidlerRuntimeEnvironment, ConfigExtender } from "../types";
```

Comment:
lets grep through the project and replace buidler -> builder (with all case combinations).

Reply:
Usually I'm using `sed` for that.

Reply:
Why later? It should be very easy to do. `sed` can handle it perfectly and it probably takes max 15min. No reason to do it later.
Otherwise there are refactoring tools for typescript which should also work with this.

Reply:
Because it will be easier to review the diff and not a single squashed commit. 
The change ss included in this PR.